inclavared/Makefile

```diff
@@ -114,7 +114,7 @@ $(App_Enclave_u_Object): app/Enclave_u.o
 	$(AR) rcsD $@ $^
 
 $(App_Name): $(App_Enclave_u_Object) $(App_SRC_Files)
-	@cd app && SGX_SDK=$(SGX_SDK) cargo build $(App_Rust_Flags)
+	@cd app && SGX_SDK=$(SGX_SDK) RUSTFLAGS="-C link-args=-Wl,-rpath=/opt/enclave-tls/lib" cargo build $(App_Rust_Flags)
```

Comment:
Use `-Wl,-rpath=*,--enable-new-dtags`. --enable-new-dtags enables the runpath instead of the depreciated rpath.